code/modules/hivemind/machines.dm

```diff
@@ -56,7 +56,7 @@
 	else if (health < max_health * 0.75)
 		to_chat(user, SPAN_WARNING("It has numerous dents and deep scratches."))
 	else if (health < max_health)
-		to_chat(user, SPAN_WARNING("It's a bit scratched and has dents."))
+		to_chat(user, SPAN_WARNING("A bit scratched and has dents."))
```

Comment:
This should stay as It's as it's a general message to the chat, not a description.